### DISCUSSION TITLE
When will support back to macOS11 or where to find the older version?

### DISCUSSION THREAD

**Discussion by catxpapa** [upscayl/upscayl#708] [Q&A] [answered]
Hello, I'm using an old macOS11 computer, I used to be able to use Upscayl, but after a certain version I can't use it anymore, I saw on the wiki that the next release will support back to macOS11, but after downloading the latest version 2.99 it still shows a minimum support for macOS12... As a temporary alternative, would like to know where I can find the previous version, please?

**Comment by NayamAmarshe** [comment]
Hi, I'm afraid you'd have to use the old versions of Upscayl for MacOS 11 or upgrade your OS

**Comment by aaronliu0130** [comment]
You can try to run/build it with the instructions under the development section of the readme. Note that you'll need to replace resources/mac/bin with a build of https://github.com/upscayl/upscayl-ncnn.

**Comment by iferni** [comment, marked as answer]
@CATxPAPA 
You can find older releases here:
https://github.com/upscayl/upscayl/releases

I am chasing the same thing.  Upscayl used to work on my PC and around 2.8.6 it broke.  Even with this trick it doesn't work:  https://github.com/upscayl/upscayl/issues/390

I am running Uspcayl on a Windows 11 laptop.  So, nope, I cannot stick an NVIDIA GPU in that machine.  :-)

**Reply by catxpapa** [reply]
Thank you, 2.8.6 is good for me. It's enough for my occasional use.
